dune-project

```diff
@@ -0,0 +1,24 @@
+(lang dune 2.7)
+
+(name pringo)
+
+(source
```

Comment:
While this is nice for upstreaming, I don't think we need more than `lang` and `name` in the `dune-project` of the port.

Reply:
`dune-project` shouldn't conflict with anything when rebasing, so since it's facilitating the update of `pringo.opam` (by temporarily generating it) and it could be useful to upstream, I'd propose to keep it?

Reply:
Sure. The reason why I objected is that it is another source of truth, so e.g if upstream changes license then it will become out of date, which is why I would avoid this if not generating OPAM files. But it doesn't hurt the port in the slightest.